### PR TITLE
2nd chance: feat query_log - optional parameter query_log_enable to enable query log

### PR DESCRIPTION
**Pull request by eumel8**

### CODE REVIEW

File: templates/named.conf.options.erb [hunk 1]
```diff
@@ -127,3 +127,25 @@ statistics-channels {
     inet <%= @statistic_channel_ip %> port <%= @statistic_channel_port %><% if @statistic_channel_allow -%> allow { <%= @statistic_channel_allow.join('; ') %>; }<% end -%>;
 };
 <% end -%>
+
+<% if @query_log_enable -%>
+logging {
+
+        channel query_logging {
+                file "/var/cache/bind/named_querylog"
```
Comment (solarkennedy) [inline templates/named.conf.options.erb]: Is there a variable you can use (`data_dir`) to remove some of the hard-coded pathiness?

Comment (eumel8) [inline templates/named.conf.options.erb]: ${data_dir} is on Debian systems '/etc/bind/zones', I wouldn't use this for log file targetting. A new variable is required for that.

**Comment by eumel8** [comment]
Don't know why Travis failed to build on Ruby 1.8. Maybe an error in the build system. Is there an option to re-trigger the build?

**Comment by solarkennedy** [comment]
It has a tendency to break because it is no longer really supported.
I'll (probably) end up removing it from the matrix.

**Pull request merged**